extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionConfiguration.java

```diff
@@ -0,0 +1,27 @@
+package io.quarkus.narayana.jta.runtime;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Inherited;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/**
+ * This annotation can be used to configure a different transaction timeout that the default one for a method or a class.
+ * It needs to be used on the entry method of the transaction.
+ */
+@Inherited
+@Target({ ElementType.METHOD, ElementType.TYPE })
+@Retention(value = RetentionPolicy.RUNTIME)
+public @interface TransactionConfiguration {
+    /** The value is used to specify that no transaction timeout is configured */
```

Comment:
Please use standard formatting for consistency.

Reply:
done